package.json

```diff
@@ -80,7 +85,8 @@
     "xml_display": "~0.1.1"
   },
   "devDependencies": {
-    "@babel/runtime": "7.0.0-beta.42",
+    "@babel/runtime": "7.0.0-beta.55",
```

Comment:
I am not sure if this is still required, the version from master should be enough (Maybe not i am not sure)